.github/workflows/deploy-tests.yml

```diff
@@ -37,14 +39,25 @@ jobs:
       AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
       S3_GIN_BUCKET: ${{ secrets.S3_GIN_BUCKET }}
       CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
-
+    with:
+      python-versions: >  # Ternary operator,  condition && value_if_true || value_if_false
+        ${{github.event.pull_request.draft == true
+           && '["3.9"]'
+           || '["3.9", "3.10", "3.11", "3.12"]'
+        }}
 
   run-live-service-tests:
```

Comment:
honestly, do we need live-service-tests or dev-tests while a PR is in draft?

In fact, we might not need them at all in the PR tests.  They could just be dailies...

Reply:
I think this makes sense. Let me think a bit.

Reply:
sounds good, separate PR is fine.